jobs/env/ci-kubernetes-e2e-gci-gce-netd.env

```diff
@@ -0,0 +1,4 @@
+### job-env
+KUBE_UP_AUTOMATIC_CLEANUP=true
```

Comment:
probably just inline these to the job definition?
this can be in a follow up though, we should probably more broadly do this @krzyzacy